Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst

```diff
@@ -321,16 +265,14 @@ Extending the database schema
 First we extend the database schema by adding the following to the file
 :file:`ext_tables.sql`:
 
-.. code-block:: sql
-
-   CREATE TABLE tt_content (
-      tx_examples_separator VARCHAR(255) DEFAULT '0' NOT NULL,
-   );
+..  literalinclude:: _AddingYourOwnContentElements/_ext_tables.sql
+    :language: sql
+    :caption: EXT:my_extension/ext_tables.sql
 
 .. tip::
 
-   Remember to do a database compare in the Install Tool after changing the
-   database schema.
+    Do a database compare in the :guilabel:`Admin Tools > Maintenance` module
```

Comment:
I suggest to mention also extension:setup command, which does this also.

Reply:
Oh? I never used it before?

Reply:
It worth to use it, especially when deploying, so new tables or fields are created automatically on the production system ;-)